### PR TITLE
Support python3

### PR DESCRIPTION
**Pull request by rgs1**
Also, release version 0.0.5.

Signed-off-by: Raul Gutierrez Segales <rgs@pinterest.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,14 @@ language: python
 sudo: false
 python:
   - "2.7"
+  - "3.4"
```
Comment (jparise) [inline .travis.yml]: 3.4 is end-of-life now so you might want to skip it.

**Comment by rgs1** [comment]
@jparise note: not ready for merge yet, but getting close. Also, when we merge lets squash.

File: thrift_tools/printer.py [hunk 2]
```diff
@@ -215,7 +215,7 @@ def report(self):
         for key, latencies in self._latencies_by_method.items():
             result = {}
             result['count'] = len(latencies)
-            result['avg'] = sum(latencies) / len(latencies)
+            result['avg'] = sum(latencies) / len(latencies) if len(latencies) > 0 else 0
```
Comment (jparise) [inline thrift_tools/printer.py]:
```suggestion
            result['avg'] = sum(latencies) / len(latencies) if latencies else 0
```

File: thrift_tools/stream_handler.py [hunk 3]
```diff
@@ -72,10 +72,10 @@ def handler(self, stream):
         # FIXME: a bit of brute force to find the start of a message.
         #        Is there a magic byte/string we can look for?
 
-        view = memoryview(context.bytes)
+        view = context.bytes
         for idx in range(0, len(context.bytes)):
```
Comment (jparise) [inline thrift_tools/stream_handler.py]: `view` and `context.bytes` are now the same thing. Simplify this a little more?

File: thrift_tools/thrift_message.py [hunk 4]
```diff
@@ -185,7 +187,7 @@ def is_binary_protocol(cls, data):
     @classmethod
     def is_json_protocol(cls, data):
         # FIXME: more elaborate parsing would make this more robust
-        return data.startswith('[1')
+        return data.tobytes().startswith(to_bytes('[1'))
```
Comment (jparise) [inline thrift_tools/thrift_message.py]:
```suggestion
        return data.tobytes().startswith(b'[1')
```

**Pull request merged**